stages/test/test_autotailor.py

```diff
@@ -51,26 +51,13 @@ def fake_input_fixture():
     }
 
 
-def schema_validate_stage_oscap_autotailor(fake_input, test_data):
-    name = "org.osbuild.oscap.autotailor"
-    version = "1"
-    root = os.path.join(os.path.dirname(__file__), "../..")
-    mod_info = osbuild.meta.ModuleInfo.load(root, "Stage", name)
-    schema = osbuild.meta.Schema(mod_info.get_schema(version=version), name)
-    test_input = fake_input
-    test_input["options"]["config"].update(test_data)
-    return schema.validate(test_input)
-
-
+@pytest.mark.parametrize("stage_module", [STAGE_NAME], indirect=True)
```

Comment:
I had a slightly different idea (maybe more messy?) in https://github.com/osbuild/osbuild/pull/1557 - it could just use  request.module.STAGE_NAME here. The downside is that it's magic, the upside less boilerplate with indirect parameters . So I guess a bit of a tricky one :)

Reply:
Sure, let's try this... :thinking: